### PR TITLE
Feature/CVSB-19222

### PR DESCRIPTION
**Pull request by PaulBenn-BJSS**
Pass-through for Dynamo events

### CODE REVIEW

File: src/services/DispatchService.ts [hunk 1]
```diff
@@ -27,39 +27,46 @@ class DispatchService {
...
     }
-    if (!removeEvent) {
-      if (!body.body.NewImage) {
-        this.logger.error(ERROR.NO_NEW_IMAGE);
-        await this.sendRecordToDLQ(record.body, target);
-        return;
-      }
-      targetRecord.body = DynamoDB.Converter.unmarshall(body.body.NewImage);
+
+    if (!dynamoEvent.dynamodb?.NewImage) {
```
Comment (bristol-mike-jones) [inline src/services/DispatchService.ts]: I'm pretty sure that REMOVE events don't contain a new NewImage.  Will this ship all events that don't contain a NewImage to the DLQ?

File: tests/unit/DispatchService.unitTest.ts [hunk 2]
```diff
@@ -132,9 +132,10 @@ describe('Dispatch Service', () => {
 
     describe('with valid REMOVE event type', () => {
       const event = {
-        eventType: 'REMOVE',
-        body: {
+        eventName: 'REMOVE',
+        dynamodb: {
           Keys: { testResultId: { S: '123' } },
+          NewImage: {}
```
Comment (bristol-mike-jones) [inline tests/unit/DispatchService.unitTest.ts]: See above.  I think that for Remove events this is omitted completely.

**Review by bristol-mike-jones — CHANGES_REQUESTED** [review]

File: src/services/DispatchService.ts (hunk 1 above)
Comment (PaulBenn-BJSS) [reply src/services/DispatchService.ts]: Addressed: https://github.com/dvsa/cvs-tsk-edh-dispatcher/pull/10/commits/36de114229059d8a15776af8eb0f10730cf073a0

File: tests/unit/DispatchService.unitTest.ts (hunk 2 above)
Comment (PaulBenn-BJSS) [reply tests/unit/DispatchService.unitTest.ts]: Addressed: https://github.com/dvsa/cvs-tsk-edh-dispatcher/pull/10/commits/6b247264e1f38437171ef46df1f25a7f4df57d33

**Review by bristol-mike-jones — APPROVED** [review]
LGTM.  We should fix up that Configuration class later though

**Review by SanjeetJ-TA — APPROVED** [review]
Happy to approve

**Pull request merged**